src/utils.rs

```diff
@@ -170,3 +175,22 @@ macro_rules! vec_of_strings {
         vec![$(String::from($str),)*] as Vec<String>
     });
 }
+
+// Create the directory ~/.config/ht/sessions/{domain} if not present.
```

Comment:
This comment isn't always correct (if `dirs::config_dir()` returns something other than (an equivalent to) `~/.config/ht/sessions/{domain}`).

```suggestion
// Create the directory which will contain sessions for `domain` if not present.
```